sdk/tests/conformance/extensions/webgl-draw-buffers.html

```diff
@@ -228,8 +228,8 @@
 
   debug("Testing drawBuffersWEBGL with default drawing buffer");
   shouldBe("gl.getParameter(ext.DRAW_BUFFER0_WEBGL)", "gl.BACK");
-  wtu.shouldGenerateGLError(gl, gl.INVALID_VALUE, "ext.drawBuffersWEBGL([])");
-  wtu.shouldGenerateGLError(gl, gl.INVALID_VALUE, "ext.drawBuffersWEBGL([gl.NONE, gl.NONE])");
+  wtu.shouldGenerateGLErrorIn(gl, [gl.INVALID_VALUE, gl.INVALID_OPERATION], "ext.drawBuffersWEBGL([])");
+  wtu.shouldGenerateGLErrorIn(gl, [gl.INVALID_VALUE, gl.INVALID_OPERATION], "ext.drawBuffersWEBGL([gl.NONE, gl.NONE])");
```

Comment:
Similarly, it looks like this one should always have been producing INVALID_OPERATION, not INVALID_VALUE, since n > 1 and the default back buffer is bound. Do you agree?